Description: Various changes to the upstream makefiles

* Don't strip binaries in Makefile.
* Install in /usr/games, not /usr/bin.
* Install cheats.bml into /usr/share/games/bsnes/, not ~/.config/bsnes.
* Name bsnes.png and bsnes.desktop explicitly.

Origin: http://anonscm.debian.org/gitweb/?p=pkg-games/bsnes.git;a=tree;f=debian/patches

--- a/bsnes/Makefile
+++ b/bsnes/Makefile
@@ -31,7 +31,7 @@
 ifeq ($(platform),x)
   # tree vectorization causes code generation errors with Linux/GCC 4.6.1
   flags += -fno-tree-vectorize
-  link += -s -ldl -lX11 -lXext
+  link += -ldl -lX11 -lXext
 else ifeq ($(platform),osx)
 else ifeq ($(platform),win)
   link += $(if $(findstring console,$(options)),-mconsole,-mwindows)
--- a/bsnes/ui/Makefile
+++ b/bsnes/ui/Makefile
@@ -76,12 +76,10 @@
 
 install:
 ifeq ($(platform),x)
-	install -D -m 755 out/$(name) $(DESTDIR)$(prefix)/bin/$(name)
-	mkdir -p ~/.config/$(name)
-	install -D -m 644 data/$(name).png $(DESTDIR)$(prefix)/share/pixmaps/$(name).png
-	install -D -m 644 data/$(name).desktop $(DESTDIR)$(prefix)/share/applications/$(name).desktop
-	cp data/cheats.bml ~/.config/$(name)/cheats.bml
-	chmod 777 ~/.config/$(name) ~/.config/$(name)/cheats.bml
+	install -D -m 755 out/$(name) $(DESTDIR)$(prefix)/games/$(name)
+	install -D -m 644 data/bsnes.png $(DESTDIR)$(prefix)/share/pixmaps/bsnes.png
+	install -D -m 644 data/bsnes.desktop $(DESTDIR)$(prefix)/share/applications/bsnes.desktop
+	install -D -m 644 data/cheats.bml $(DESTDIR)$(prefix)/share/games/bsnes/cheats.bml
 endif
 
 uninstall:
